src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -80794,6 +80794,24 @@ class Program
                 Diagnostic(ErrorCode.WRN_NullAsNonNullable, "null").WithLocation(4, 23));
         }
 
+        [Fact]
+        [WorkItem(26628, "https://github.com/dotnet/roslyn/issues/26628")]
+        [WorkItem(33394, "https://github.com/dotnet/roslyn/issues/33394")]
+        public void ImplicitConstructor_03()
```

Comment:
>ImplicitConstructor_03 [](start = 20, length = 22)

What does "Implicit" refer to for this scenario?

Reply:
"Implicit" refers to the static constructor, although I'll admit it's confusing.